pennylane/devices/default_qubit_autograd.py

```diff
@@ -92,6 +92,7 @@ class DefaultQubitAutograd(DefaultQubit):
         "CRX": autograd_ops.CRX,
         "CRY": autograd_ops.CRY,
         "CRZ": autograd_ops.CRZ,
+        "CRot": autograd_ops.CRot,

```

Comment:
Noticed this was missing due to tests suddenly failing in the new backprop default!